README.md

```diff
@@ -1,43 +1,65 @@
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
 
-## Usage
+Baseline image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
 
-Follow these instructions if you are running docker *outside* of Google Compute Engine:
+Imaage includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
 
-    # Get the cloud sdk image:
-    $ docker pull google/cloud-sdk
 
-    # Auth & save the credentials in gcloud-config volumes:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud init
+## Dockerfile
 
-    # If you would like to use service account instead please look here:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud auth activate-service-account <your-service-account-email> --key-file /tmp/your-key.p12 --project <your-project-id>
+To use the prebuilt **Alpine** based image, see
 
-    # Re-use the credentials from gcloud-config volumes & run sdk commands:
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud info
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud components list
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud compute instances list
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gsutil ls
+* [https://hub.docker.com/r/google/cloud-sdk/](https://hub.docker.com/r/google/cloud-sdk/)
 
-If you are using this image from *within* [Google Compute
-Engine](https://cloud.google.com/compute/). If you enable a Service Account with
-the necessary scopes, there is no need to auth or use a config volume:
 
-    # Get the cloud sdk image:
-    $ docker pull google/cloud-sdk
+```bash
+docker run -ti google/cloud-sdk:latest
+```
 
-    # Just start using the sdk commands:
-    $ docker run --rm -ti google/cloud-sdk gcloud info
-    $ docker run --rm -ti google/cloud-sdk gcloud components list
-    $ docker run --rm -ti google/cloud-sdk gcutil listinstances
-    $ docker run --rm -ti google/cloud-sdk gsutil ls
+or via tagged version of the SDK:
+
+```bash
+docker run -ti google/cloud-sdk:153.0.0
+```
+
+>> **NOTE:** you may need to install the optional components:
+```
+
+For example, for python GAE support, you would need to extend the base image:
+
+```dockerfile
+FROM google/cloud-sdk
+RUN gcloud components install app-engine-python
+```
+
+or for java
+```
+FROM google/cloud-sdk
+RUN apk --update add openjdk7-jre
+RUN gcloud components install app-engine-java
+WORKDIR /apps
+```
+
+For a volume mount using credentials:
```

Comment:
This line and the rest of the section has nothing to do with app-engine-java and app-engine-python. I figured you're showing _all users_  how to use the image the here, which is super important. You should move this up.